### ISSUE TITLE
ui: improved auto-renew popup

### ISSUE THREAD

**Issue by lrstanley** [hashicorp/vault#4840]
Currently, when being inactive on the webui, a user will see the following popup:
![image](https://user-images.githubusercontent.com/1847365/41932072-f77f67b6-794d-11e8-84d7-d8646e09b49f.png)

This blocks a good portion of the screen, and I'm not quite sure what the use is in having it in a popup like this. Were there design/security considerations for making it a popup, and making users manually click a button?

I feel, from a UI perspective, if there was a security consideration, to keep it, but minimize it in the view (terrible mockup, but you get the idea):
![image](https://user-images.githubusercontent.com/1847365/41932951-7eae0222-7950-11e8-830d-368beac32415.png)

Or alternatively, overlay a yellow warning icon on top of the user dropdown on the far right, and within the dropdown, adding a 'Continue auto-renewing'. etc.

If there was no specific security concern to ensure the user manually re-enables auto-renew, it may be better to just re-enable auto-renew in the background without user interaction -- and add a small status in the nav saying `not auto-renewing token due to inactivity`, when they are inactive. It would be useful to add some kind of 'token expiring soon' in place of that status text, when the token gets close to it's expiration period.

In short: right now, it can be confusing to users logging into vault, and can be irritating when switching between windows over and over again. Furthermore, I currently see a ~5s delay when clicking the enable button, before the popup disappears.
